docs/spec/message-store-service-api.yml

```diff
@@ -11,22 +11,97 @@ tags:
     description: >-
       Represents blockchain events in the canonical data format used by
       Relayers.
-  - name: events
-    description: Represents raw blockchain events emitted by a network.
 schemes:
   - https
 paths:
+  /messages:
+    get:
+      tags:
+        - messages
+      summary: Retrieve messages that match the provided filter criteria.
+      description: >-
+        Returns message that match the filter criteria passed as query
+        parameters.
+      produces:
+        - application/json
+      parameters:
+        - name: networkId
+          in: query
+          description: ID of network in which the event was generated
+          required: true
+          type: string
+        - name: blockNumber
+          in: query
+          description: The block number that contains this event
+          required: true
+          type: string
+        - name: txIndex
```

Comment:
How about adding filter message by transaction hash? Might be easier for user to query

Reply:
sorry for the delayed response @wcgcyx
Yes, transaction hash makes sense, I'll add it